backend/src/game/cards/card_type.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class CardType(Enum):
```

Comment:
All caps for the enum words, KEEP, etc

Reply:
Then we should update the locations enum 
`
from enum import Enum


class Locations(Enum):
    OUTSIDE = 0
    TOKYO = 1

`

Reply:
@borisv13  correct me if I'm wrong, but does this mean in Python enums should be named like classes? https://www.python.org/dev/peps/pep-0435/

Reply:
https://stackoverflow.com/questions/21359130/should-3-4-enums-use-upper-case-with-underscores
https://docs.python.org/3/library/enum.html#creating-an-enum

I thought I did do Locations as all caps... am I crazy?

Reply:
OH! I misunderstood. I thought you meant LOCATION.TOKYO not Location.TOKYO.  Now I get it. I'll fix that

Reply:
Fixed it